apps/web/lib/signup/getServerSideProps.tsx

```diff
@@ -24,8 +24,9 @@ const querySchema = z.object({
 
 export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
   const prisma = await import("@calcom/prisma").then((mod) => mod.default);
+  await ssrInit(ctx);
```

Comment:
can this be removed too?

Reply:
No they seems to be the only way translations load on signup 🤔